### PR TITLE
Copyright

### PR DESCRIPTION
**Pull request by lsat12357**

### CODE REVIEW

File: oregon/templates/home.html [hunk 1]
```diff
@@ -38,54 +38,9 @@
...
...proxtext=&amp;proxdistance=5&amp;rows=20&amp;searchType=advance" title="Oregon City newspapers" alt="Oregon City newspapers" /><area shape="rect" coords="141,74,198,89" href="http://oregonnews.uoregon.edu/search/pages/results/?lccn=sn83025138&amp;lccn=sn84022673&amp;lccn=sn85042444&amp;lccn=sn83045780&amp;lccn=sn00063676&amp;lccn=sn83045782&amp;date1=1848&amp;date2=1922&amp;date1=&amp;date2=&amp;sequence=&amp;ortext=&amp;andte" title="Portland newspapers" alt="Portland newspapers" /><area shape="rect" coords="47,83,112,101" href="http://oregonnews.uoregon.edu/search/pages/results/?lccn=sn99063850&amp;dateFilterType=yearRange&amp;date1=1846&amp;date2=1922&amp;ortext=&amp;andtext=&amp;phrasetext=&amp;proxtext=&amp;proxdistance=5&amp;rows=20&amp;searchType=advanced&amp;sort=date" title="Tillamook newspapers" alt="Tillamook newspapers" /><area shape="rect" coords="100,39,166,55" href="http://oregonnews.uoregon.edu/search/pages/results/?lccn=sn97071109&lccn=sn93051654&lccn=2004260421&lccn=2004260419&dateFilterType=yearRange&date1=1846&date2=2013&ortext=&andtext=&phrasetext=&proxtext=&proxdistance=5&rows=20&searchType=advanced" title="St. Helens newspapers" alt="St. Helens newspapers" />
```
Comment (jechols) [inline oregon/templates/home.html]: I don't understand what this change has to do with the copyright issue.  Even so, at the least, break up the text into one line per `<area>` tag like they used to be - a single line that's 16000 characters is really awful to edit or diff.

**Comment by jechols** [comment]
I think this PR is missing some schema-related file or files.  I don't remember how you handled the migration issue, but I thought we needed something that told django how to create the new field and table.

File: oregon/templates/home.html (hunk 1 above)
Comment (jechols) [inline oregon/templates/home.html]: Actually I think I see what changed here.  The formatting definitely needs a fix, as it's really only a minor change to a single `<area>` tag.  But I'd also like to see the "map changes" commit pulled out of this PR and put into a new PR or just master directly.  Unless I'm totally crazy, that commit is an unrelated fix to the homepage map.

**Comment by lsat12357** [comment]
oh oops. not sure how the map change ended up in this branch. wasn't intentional.

**Comment by lsat12357** [comment]
re the migration-- I changed the models as in the commit, then ran 
django-admin.py schemamigration chronam.core --auto
django-admin.py migrate chronam.core
then fired up mysql and manually added copyright vals to the table.

**Comment by jechols** [comment]
It looks like django generated a file - check out your "git status".  There's some core migration file that I'm pretty sure we want in git.

**Comment by jechols** [comment]
:+1:
